addons/binding/org.openhab.binding.oceanic/README.md

```diff
@@ -76,3 +76,29 @@ Number oceanicConsComplete "volume all time is [%d]"(oceanic) {channel="oceanic:
 Number oceanicConsUntreated "volume untreated is [%d]"(oceanic) {channel="oceanic:softener:s1:consumptionuntreated"}
 Number oceanicConsLastWk "volume last week is [%d]"(oceanic) {channel="oceanic:softener:s1:consumptionlastweek"}
 ```
+
+## Note about nrjavaserial
```

Comment:
Maybe rename to known issues, the users are most likely not aware of `nrjavaserial`